test/ruby_lsp_rails/hover_test.rb

```diff
@@ -181,6 +167,10 @@ class ActiveRecord::Base
 
       def hover_on_source(source, position)
         with_server(source) do |server, uri|
+          while RubyLsp::Addon.addons.first.instance_variable_get(:@client).instance_of?(RubyLsp::Rails::NullClient)
```

Comment:
We're doing the same in https://github.com/Shopify/ruby-lsp-rails/pull/303 but this should be cleaned up.